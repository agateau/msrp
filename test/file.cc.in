// bb
/// \file $ID$ 
// aa

void a()
{
}



// my file: $ID$
