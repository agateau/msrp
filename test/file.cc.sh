#! /bin/sh

echo "TESTING \f option: "

cp -f file.cc.in file.cc

echo "forward test:"
$MSRP -f -d "[$]ID[$]" "\\f" file.cc

cmp file.cc file.cc.out || exit 1

echo "PASSED!"

echo "backward test:"
$MSRP -f -d "\f" "\$ID\$" file.cc

cmp file.cc file.cc.in || exit 1

echo "PASSED!"

rm file.cc



