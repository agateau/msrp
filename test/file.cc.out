// bb
/// \file file.cc 
// aa

void a()
{
}



// my file: file.cc
